README.md

```diff
@@ -11,36 +11,45 @@
 | Docs | [![gh-pages](https://img.shields.io/github/last-commit/holoviz/holoviz/gh-pages.svg)](https://github.com/holoviz/holoviz/tree/gh-pages) [![site](https://img.shields.io/website-up-down-green-red/http/holoviz.org.svg)](http://holoviz.org) [![dev-site](https://img.shields.io/website-up-down-green-red/https/pyviz-dev.github.io/holoviz.svg?label=dev%20website)](https://pyviz-dev.github.io/holoviz/)  |
 | Status Dashboard | [![](https://img.shields.io/website-up-down-green-red/http/status.holoviz.org.svg?label=status-dashboard)](http://status.holoviz.org/) |
 | Binder  | [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/holoviz/holoviz/HEAD?labpath=examples%2Ftutorial%2F00_Setup.ipynb) |
-| Code of Conduct | [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)|
+| Code of Conduct | [![Based on Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)|
 
 
-## What is it?
+## What is HoloViz?
 
-This repository provides examples, demos, and training materials
-documenting how to solve visualization problems using Python
-tools that work with web browsers, including
-[Bokeh](https://bokeh.org),
-[HoloViews](https://holoviews.org),
-[GeoViews](http://geoviews.org),
-[Matplotlib](https://matplotlib.org),
-[Datashader](https://datashader.org),
-[hvPlot](https://hvplot.holoviz.org),
-[Panel](https://panel.holoviz.org), and
-[Param](https://param.holoviz.org).
+HoloViz provides high-level tools that are designed to work together to solve the entire problem of visualization, from conducting exploratory data analysis to deploying complex dashboards.
 
-It includes a shared conda environment that ensures all these tools work together,
-facilities for downloading sample data for use with these tools, and provides
-starting points for solving your own visualization problems.
+The core HoloViz projects are:
 
+- [hvPlot](https://hvplot.holoviz.org): interactive plotting built on Bokeh, Matplotlib, and Plotly
+- [HoloViews](https://holoviews.org): declarative high-level data description and plotting
+- [GeoViews](http://geoviews.org): geographic data extension for HoloViews
+- [Datashader](https://datashader.org): server-side rendering of large datasets
+- [Colorcet](https://colorcet.holoviz.org/): perceptually based colormapping
+- [Panel](https://panel.holoviz.org): interactive apps and dashboards
+- [Lumen](https://lumen.holoviz.org/): no-code apps and dashboards
+- [Param](https://param.holoviz.org): declarative parameter-based coding
```

Comment:
Maybe re-use or align the order and definitions with what is on https://holoviz.org/background.html?

Reply:
Thanks for the feedback. I've updated the order to align with the background page, but I think we can improve upon the project descriptions that are on that page (for instance, someone should not have to know what HoloViews, GeoViews, Panel, Pandas, and Xarray are in order to understand the value proposition for hvPlot). I took another shot at writing these descriptions, trying to keep the language as short and simple as possible. Let me know if would suggest further changes

Reply:
That's much better thanks!